server/models/schema.sql

```diff
@@ -2052,4 +2052,12 @@ CREATE TABLE `department`(
   FOREIGN KEY (`enterprise_id`) REFERENCES `enterprise` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
+
+CREATE TABLE `tags`(
+  `uuid` BINARY(16) NOT NULL PRIMARY KEY,
+  `name` VARCHAR(100) NOT NULL,
+  UNIQUE KEY  (`name`)
```

Comment:
It looks like tags are supposed to be unique - could you add a test to make sure you cannot add the same name tag twice?
It would be best as an integration test.